test/unit/range/views/view_interleave_test.cpp

```diff
@@ -87,6 +89,7 @@ TEST(view_interleave, concepts)
     EXPECT_FALSE((std::ranges::output_range<decltype(v2), dna4>));
 }
 
+#if 0 // blocked by https://github.com/ericniebler/range-v3/issues/1320
```

Comment:
There's a card?
Was there a similar reason for the other `#if 0`

Reply:
There will be, see above.